src/app/connected-component/connected.component.ts

```diff
@@ -46,7 +54,9 @@ export class ConnectedComponent extends BaseComponent implements OnInit, OnDestr
             return;
         }
         console.info(this.componentID, 'is shutting down')
-        this.connectionService.removeConnection(this.componentID);
+        if (this.connected) {
```

Comment:
This should also set this.connected to false

Reply:
added